scripts/gen_offset_header.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2017 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from elftools.elf.elffile import ELFFile
+from elftools.elf.sections import SymbolTableSection
+import argparse
+import sys
+
+def gen_offset_header(input_name, input_file, output_file):
+  obj = ELFFile(input_file)
+
+  symtab = obj.get_section_by_name('.symtab')
+  if not symtab:
```

Comment:
I suggest throwing an exception in this and the below isinstance check if they fail